cosmic_ray/operators/remove_decorator.py

```diff
@@ -0,0 +1,27 @@
+import ast
+
+from .operator import Operator
+
+
+class RemoveDecorator(Operator):
+    """An operator that removes each of the non standard decorators."""
+    regular_decorators = ["classmethod", "staticmethod", "abstractmethod"]
```

Comment:
This should be a `frozenset` with an all-caps name to indicate that it's a "constant":
```
REGULAR_DECORATORS = frozenset(("classmethod", "staticmethod", "abstractmethod"))
```